docs/guide/release-notes.asciidoc

```diff
@@ -0,0 +1,261 @@
+[[release-notes]]
+== Release notes
+
+* <<rn-7110>>
```

Comment:
I'm worried about this link system because there's no way to distinguish `7.1.10` and `7.11.0`, can we use `-` delimiters between the numbers?

Reply:
Addressed via b7e33a8